### PR TITLE
Update my branch with swiping and calendar functionality

### PR DESCRIPTION
**Pull request by ShonjoyBhattacharyya**

### CODE REVIEW

**Review by allisonfain08 — APPROVED** [review]
looks incredible, will review now

File: altsfw-USETHISONE/altsfw/CalendarViewController.swift [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import UIKit
+import EventKit
+import EventKitUI
+
+class CalendarViewController: UIViewController, EKEventEditViewDelegate {
+    func eventEditViewController(_ controller: EKEventEditViewController, didCompleteWith action: EKEventEditViewAction) {
+        controller.dismiss(animated: true, completion: nil)
+    }
+    
+    let eventStore = EKEventStore()
```
Comment (MijaKennedy) [inline altsfw-USETHISONE/altsfw/CalendarViewController.swift:17]: This looks great, will help us with calendar user stories

File: altsfw-USETHISONE/altsfw/QuizViewController.swift [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import UIKit
+import SwiftUI
+
+var swipeTracker = SwipeTracker()
+class QuizViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+        let swipeRight = UISwipeGestureRecognizer(target: self, action: #selector(swipeFunc))
```
Comment (MijaKennedy) [inline altsfw-USETHISONE/altsfw/QuizViewController.swift:17]: great work, this looks good to me. can add an enum for dummy styles (or client specific if received)

**Review by MijaKennedy — APPROVED** [review]
looks good to me, great work!

**Pull request merged**